### ISSUE TITLE
Fix copy() in RasterPacked heirs

### ISSUE THREAD

**Issue by davidhesselbom** [cogneco/ooc-kean#533]
With #504, `RasterMonochrome copy()` will work, but the other variants won't. Also, the `copy` function in `RasterImage` doesn't do what it should.

**Comment by marcusnaslund** [comment]
To clarify, the problem is that `_buffer` never is initialized. For reference, here is my comment in #504 about this:
The constructor in `RasterPacked` used by `copy` is:
 ```ooc
init: func ~fromOriginal (original: RasterImage)
```
We cannot do anything with the buffer in `original` here unless it explicitly is set to be of type `RasterPacked`. But for it to be, we also have to update the appropriate constructor in each subclass since they also expect a `RasterImage`:
```ooc
RasterMonochrome: class extends RasterPacked {
// ...
init: func ~fromRasterImage (original: RasterImage) { super(original) }
// ...
```
Or, add a new `init ~fromRasterPacked`.

**Comment by davidhesselbom** [comment]
Yeah, ok, so another constructor should be used, because `~fromOriginal` should be able to take any `RasterImage`, including `RasterYuv420Semiplanar`, for instance.

**Comment by davidhesselbom** [comment]
*OR* it should only take a `This`, because converting from other formats is what the `convertFrom` function is for.

**Comment by marcusnaslund** [comment]
Resolved by #504
